doc/user/overview.rst

```diff
@@ -239,7 +239,15 @@ The indicators have the following semantics:
 * :mark:`CP` - control plane only (i.e. BGP route server / route reflector)
 * :mark:`N` - daemon/feature not supported by operating system
 
-.. Known Kernel Issues:
+.. note::
+
+   At this time, FRR offers partial support for some of the routing
+   protocol extensions that can be used with MPLS-TE. FRR does not
+   support a complete RSVP-TE solution currently.
```

Comment:
I think we can omit the note here. OSPF and Zebra docs are enough imo.

Reply:
got it - removing this note